hm_1/tasks/task_07_03.py

```diff
@@ -15,12 +15,26 @@
      [x, x, x]]
      Return value should be "x wins!"
 """
-import numpy as np
+
+
+def transponse(board):
+    transponse_list = []
+    for i in range(len(board)):
+        transponse_list.append([row[i] for row in board])
+    return transponse_list
```

Comment:
Zip accepts iterators, so you can convert this code to generator:

```suggestion
def transponse(board):
    for i in range(len(board)):
        yield [row[i] for row in board]
```